### PR TITLE
Update Storage Blob SDK to T2

### PR DESCRIPTION
**Pull request by zzhxiaofeng**
SDK Updated List:
Azure Storage Blob

### CODE REVIEW

**Comment by ramya-rao-a** [comment]
@ljian3377, @EmmaZhu, Can one of you review this PR please?

File: index.js [hunk 1]
```diff
@@ -1,149 +1,177 @@
...
-            if (err) {
-                reject(err);
-            } else {
-                resolve({ message: `${data.entries.length} containers`, containers: data.entries });
-            }
-        });
-    });
+  return new Promise(async (resolve, reject) => {
+    try {
+      const data = (await blobServiceClient.listContainers().byPage().next())
```
Comment (ljian3377) [inline index.js:14]: This only listed one page of the containers. Do you also want to illustrate the usage of the continuation token?

Comment (zzhxiaofeng) [reply index.js]: Ok, I have updated the code. Please have a look if the question is resolved.

**Comment by ramya-rao-a** [comment]
Thanks for updating @zzhxiaofeng !

@ljian3377, Can you take another look?

**Review by ljian3377 — APPROVED** [review]

**Comment by ramya-rao-a** [comment]
@zzhxiaofeng On second thoughts, we should probably not update this sample at all.
 The readme already has a warning saying that this sample uses the older package and points the reader to https://github.com/Azure-Samples/storage-blobs-node-quickstart which has both v10 and v12 package samples.

> WARNING You are viewing a tutorial for the legacy version of the Azure Storage Node.js SDK. Please go here to view the latest JavaScript SDK (v10).

We should close this PR and open a new one to update the warning to include v12

**Comment by zzhxiaofeng** [comment]
@ramya-rao-a  Ok, We don't close the PR and use the PR to update warning as you said.

File: README.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ urlFragment: storage-blobs-node-quickstart
 # Storage Blobs Quickstart
 
 > **WARNING** 
-> You are viewing a tutorial for the legacy version of the Azure Storage Node.js SDK. Please go [here](https://github.com/Azure-Samples/azure-storage-js-v10-quickstart) to view the latest JavaScript SDK (v10).
+> You are viewing a tutorial for the legacy version of the Azure Storage Node.js SDK. Please go [here](https://github.com/Azure-Samples/azure-storage-js-v10-quickstart) to view the latest JavaScript SDK (v10 and v12).
```
Comment (ramya-rao-a) [inline README.md:14]: We now have a standard template for adding such messages in the sample repo. Please use this format in future PRs as well.

```suggestion
> Samples in this repository use older package `azure-storage` for Azure Storage Blobs.
We recommend that you refer to the [samples that use the new package](https://github.com/Azure-Samples/azure-sdk-for-js-storage-blob-upload-download) `@azure/storage-blob` instead.
```

Comment (zzhxiaofeng) [reply README.md]: Ok, Readme is updated, please have a review.

**Review by ramya-rao-a — APPROVED** [review]

**Review by ramya-rao-a — APPROVED** [review]

**Comment by XiaoningLiu** [comment]
I don't have permission to merge PR to this repo. 
@georgewallace can you help take a look?

**Comment by ramya-rao-a** [comment]
@jongio can help with merging PRs

**Pull request merged**